fe/fe-core/src/main/java/com/starrocks/qe/SetDefaultRoleExecutor.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.starrocks.qe;
+
+import com.starrocks.analysis.UserIdentity;
+import com.starrocks.common.UserException;
+import com.starrocks.privilege.PrivilegeException;
+import com.starrocks.privilege.PrivilegeManager;
+import com.starrocks.sql.ast.SetDefaultRoleStmt;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetDefaultRoleExecutor {
```

Comment:
Since we have added the default role feature in new privilege framework, and in old version user can also have default role, do we need to consider the upgrade behavior for default role?